wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -13,7 +13,9 @@ end
 
 namespace :wagn do
   desc "create a wagn database from scratch"
-  task :create do
+  task :recreate do
```

Comment:
I think you used *deck apps* in a more recent comment where it seemed to be in contrast to an *app deck*, but I think you are right *engine deck* maybe better (or *rails deck* which could be considered an alternate for *engine deck*).  Those seem fine for the boolean methods.

My thinking was that eventually, there would be just the decko and card gems.  The wagn gem is already getting pretty empty.  In my view, we only split them (wagn and decko-rails) as a transitional thing because we aren't ready to re-brand the whole thing yet.  So, we may find a good reason to keep this split, but I think the final state will be that you install the decko gem, then you can do either 1) run decko commands to generate an app deck, or 2) add decko to your Gemfile and run rails generator commands to add the models and engine to your app.